certified-connectors/Nextset/apiProperties.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
@taiSateraito And you have uploaded a swagger file in the apiProperties.json file.